From 45fc42c5c980e2d5b755ed24a6983f44798d958e Mon Sep 17 00:00:00 2001
From: WANG Chao <chaowang@redhat.com>
Date: Tue, 10 Jun 2014 14:11:27 +0900
Subject: [PATCH] [PATCH] Fix Makefile for eppic_makedumpfile.so build.

When libeppic isn't installed on a standard location, building
eppic_makedumpfile.so with -leppic directly doesn't work.

Add LDFLAGS to build arguments, so that one can pass LDFLAGS="-Ldir
-Idir" to tell where to search for libeppic library and its header
files.

For example, if eppic source is installed on the same directory level
with makedumpfile as the following:

makedumpfile
 |--- arch
 +--- eeppic_scripts
eppic
 |--- applications
 +--- libeppic

After compiling libeppic, one can use the following command to build
eppic_makedumpfile.so:

make LDFLAGS="-I../eppic/libeppic -L../eppic/libeppic" eppic_makedumpfile.so

Signed-off-by: WANG Chao <chaowang@redhat.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makedumpfile-1.3.5/Makefile b/makedumpfile-1.5.3/Makefile
index d87638b..0cc07ef 100644
--- a/makedumpfile-1.3.5/Makefile
+++ b/makedumpfile-1.3.5/Makefile
@@ -87,7 +87,7 @@ makedumpfile: $(SRC_BASE) $(OBJ_PART) $(OBJ_ARCH)
 	gzip -c ./makedumpfile.conf.5 > ./makedumpfile.conf.5.gz
 
 eppic_makedumpfile.so: extension_eppic.c
-	$(CC) $(CFLAGS) -shared -rdynamic -o $@ extension_eppic.c -fPIC -leppic -ltinfo
+	$(CC) $(CFLAGS) $(LDFLAGS) -shared -rdynamic -o $@ extension_eppic.c -fPIC -leppic -ltinfo
 
 clean:
 	rm -f $(OBJ) $(OBJ_PART) $(OBJ_ARCH) makedumpfile makedumpfile.8.gz makedumpfile.conf.5.gz
-- 
1.9.3

